submission/launch.sh

```diff
@@ -13,23 +13,18 @@ chmod +x script.sh
 vagrant up
 (
 set +e
-vagrant ssh -- 'cd /vagrant; ./script.sh' 1> result.out 2> result.err
+vagrant ssh -- 'cd /vagrant; ./script.sh' &> result.out
 echo $? > result.exit_code
 )
 
 exit_code=$(cat result.exit_code)
 base64 result.out > stdout.tmp
 cat stdout.tmp | tr -d '\n' > stdout
 
-base64 result.err > stderr.tmp
-cat stderr.tmp | tr -d '\n' > stderr
-
-
 jq -n \
     --rawfile out stdout \
-    --rawfile err stderr \
     --arg code $exit_code \
-    '{stdout: $out, stderr: $err, exit_code: $code,}' > out.json
+    '{stdout: $out, exit_code: $code,}' > out.json
```

Comment:
This will fail because `acs-interface` [expects the `stderr` parameter](https://github.com/vmck/acs-interface/blob/master/interface/views.py#L163). We need to make it optional: `stderr = utils.decode(options.get('stderr', ""))`. Please amend https://github.com/vmck/acs-interface/pull/115.

Reply:
Done - I changed in the other PR

Reply:
LE: I removed the stderr field